lucene/core/src/java/org/apache/lucene/index/ExitableDirectoryReader.java

```diff
@@ -117,6 +117,7 @@ public CacheHelper getCoreCacheHelper() {
     private final PointValues in;
     private final QueryTimeout queryTimeout;
 
+    /** Constructor **/
```

Comment:
My comment
> But isn't it worse now? precommit is satisfied, but nothing more than (obvious) noise was added.

was for this line.

Reply:
My bad, I though I used this class in tests, but I didn't. I've turned it `private` in 6f48d4f851b4eccf4c3d8c4cfbe6d540e13b4d85, thanks :+1: